controllers/infinispan_controller.go

```diff
@@ -848,9 +863,36 @@ func IsUpgradeRequired(infinispan *infinispanv1.Infinispan, kube *kube.Kubernete
 	return upgradeRequired(infinispan, podList)
 }
 
-func PodList(infinispan *infinispanv1.Infinispan, kube *kube.Kubernetes, ctx context.Context) (*corev1.PodList, error) {
+func PodListStatefulSet(statefulSetName, namespace, infinispanName string, kube *kube.Kubernetes, ctx context.Context) (*corev1.PodList, error) {
```

Comment:
Lets move this and `PodList` to `pod.go` to reduce the clutter in `infinispan_controller.go`.